.travis.yml

```diff
@@ -53,7 +53,7 @@ script:
 - if [[ "$TEST_ENV" == "conda" ]];
   then
     pytest &&
-    rm -f docs/rhg_compute_tools.*.rst &&
+    rm -f docs/rhg_compute_tools*.rst docs/modules.rst &&
```

Comment:
Actually excluding docs/modules is intentional. This allows us to have our docs section be called "API" instead of "rhg_compute_tools" which is confusing. If you could roll back this change and the changes to `docs/modules.rst` ([here](https://github.com/RhodiumGroup/rhg_compute_tools/pull/71/files#diff-048b1487c1c99241f36bcfebe3679f05R1)) that would be great!

Reply:
done